### ISSUE TITLE
Convert to native keras

### ISSUE THREAD

**Issue by bretthandrews** [bretthandrews/CapsNet-Keras#3]
There are two instances of TF code:
- tf.softmax
- tf.zeros
